src/components/MapView/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import style from './style.css';
+class MapView extends Component 
+{
+
+    mapfunc = () => {
+        var mapboxgl = require('mapbox-gl/dist/mapbox-gl.js');
+ 
+        mapboxgl.accessToken = 'pk.eyJ1IjoiZGtwMTkwMyIsImEiOiJjazhpdTB6dGEwOXpvM2xxbWFhY25vdHpuIn0.sFq07QKbSsb3vSOBGJZ8lg';
+        var map = new mapboxgl.Map({
+        container: 'map',
+        style: 'mapbox://styles/mapbox/streets-v11'
+        }
+        );
+
+    }
+
+	render() {
+        
+        
+		return(
+			<div id='map'>
+				<h2>
+					{this.mapfunc}
+					</h2>
+					
+				<p>
+					Maps ought to be coming here
+				</p>
+			</div>
+        )
+        
+	}
+}
+
+export default MapView;
```

Comment:
cool I think it works , I did not test it , we have CI /CD setup , so before merging this can you make a request to /property and display all of them on map with a marker that will be a step forward where we want to go, and don't really seems like a half finished work